source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js

```diff
@@ -713,6 +713,12 @@
     };
 
     layoutViewModel = new LayoutViewModel();
+    window.onbeforeunload = function () {
+        if (viewmodel.isDirty()) {
```

Comment:
(Very very minor and opinable) Since you are touching a `viewmodel`'s property, what do you think about to move it near to `viewmodel` declaration? for example after `viewmodel.load();`